custom-seldon-server/README.md

```diff
@@ -14,6 +14,7 @@
 2. pytorch
 3. sklearn
 4. huggingface
+5. base
```

Comment:
Curious about base is used to?

Reply:
The base custom Seldon server uses to build up their custom Seldon server environment by themselves in PrimeHub Images.